# For the info to be sent in and around for the low pass filter

time stamp
float64 x
float64 y
float64 ang_vel
